src/test/java/org/gitlab4j/api/TestGitLabApi.java

```diff
@@ -77,6 +77,7 @@ public void testGetVersion() throws GitLabApiException {
     @Test
     public void testProxyConnection() throws GitLabApiException {
         assumeTrue(TEST_PROXY_URI != null && TEST_PROXY_USERNAME != null && TEST_PROXY_PASSWORD != null);
+        assumeTrue(TEST_PROXY_URI.length() > 0 && TEST_PROXY_USERNAME.length() > 0 && TEST_PROXY_PASSWORD.length() > 0);
```

Comment:
Suggest using `TEST_PROXY_URI.isEmpty()` for length checks.

Reply:
```TEST_PROXY_URI.length() > 0```
        vs
```!TEST_PROXY_URI.isEmpty()```


When checking for truth, I prefer to use a statement that results in true, not something that has to be negated to produce true.  It's just a personal preference and is used throughout GitLab4J, so to change it in these instances would require a sweep through all the code to be consistent.

Reply:
The reasoning sounds good to me.

Reply:
@lamdaV 
FYI, when checking for empty instead of not empty I should be using isEmpty(), so I have gone through all the tests and replaced str.size() == 0 with str.isEmpty(), just thought I would let you know.